CHANGELOG.md

```diff
@@ -17,7 +17,9 @@ _None_
 
 ### Bug Fixes
 
-_None_
+* Build: Fixed the `rake cli:install` command and with it the Homebrew formula
+  [Liquidsoul](https://github.com/liquidsoul)
+  [#1030](https://github.com/SwiftGen/SwiftGen/issues/1007)
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/67caedd575ca1566c177bfe96325a7e991899880/CHANGELOG.md#L22"></span>Link text is #1030 but links points to SwiftGen/SwiftGen#1007.</td>
    </tr>
  </tbody>
</table>